tests/Behat/Page/Shop/Order/ShowPage.php

```diff
@@ -12,4 +13,38 @@ public function countCreditMemos(): int
     {
         return count($this->getDocument()->findAll('css', '#credit-memos tbody tr'));
     }
+
+    public function downloadFirstCreditMemo(): void
+    {
+        $creditMemo = $this->getFirstCreditMemo();
+        $creditMemo->clickLink('Download');
+    }
+
+    public function isPdfFileDownloaded(): bool
```

Comment:
It should not be implemented here, we have `Tests\Sylius\RefundPlugin\Behat\Context\Element\PdfDownloadElement` (which in fact should be moved to `Tests\Sylius\RefundPlugin\Behat\Element\PdfDownloadElement`)